templates/index.hbs

```diff
@@ -1,125 +1,131 @@
 {{#if newNodeVersionAvailable}}
 <div class="alert alert-dismissible alert-warning">
-  <button type="button" class="close" data-dismiss="alert">&times;</button>
-  <strong>Heads up!</strong> A new Nano Node version
-  <strong>{{newNodeVersionAvailable}}</strong> is available.
-  <a href="https://github.com/nanocurrency/raiblocks/releases/latest"
-    target="_blank">Download it here!</a>
+    <button type="button" class="close" data-dismiss="alert">&times;</button>
+    <strong>Heads up!</strong> A new Nano Node version
+    <strong>{{newNodeVersionAvailable}}</strong> is available.
+    <a href="https://github.com/nanocurrency/raiblocks/releases/latest"
+       target="_blank">Download it here!</a>
 </div>
 {{/if}}
 
 <div class="row">
-  <div class="col">
-    <h2>Node Account</h2>
-    <ul class="list-group">
-      <li class="list-group-item active d-flex justify-content-between align-items-center">
-        <span class="mr-3">Address</span>
-        <a href="{{nanoNodeAccountUrl}}" target="_blank" class="truncate">{{nanoNodeAccount}}</a>
-        <button id="copyAccount" class="btn btn-sm btn-link" data-clipboard-text="{{nanoNodeAccount}}" title="Copy">
-          <i class="fas fa-clipboard fa-lg"></i>
-        </button>
-      </li>
-    </ul>
-  </div>
+    <div class="col">
+        <h2 class="n-h2-theme">Node Account</h2>
```

Comment:
Having Node Account as h2 and the other heading as h3 looks weird to me: https://imgur.com/a/1sbYBUs

Reply:
I personally prefer it, but that is an easy change